Tests/ML/histopathology/datasets/test_tcga_prad_dataset.py

```diff
@@ -1,34 +0,0 @@
-#  ------------------------------------------------------------------------------------------
-#  Copyright (c) Microsoft Corporation. All rights reserved.
-#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
-#  ------------------------------------------------------------------------------------------
-
-import os
-
-import pytest
-
-from InnerEye.ML.Histopathology.datasets.default_paths import TCGA_PRAD_DATASET_DIR
-from InnerEye.ML.Histopathology.datasets.tcga_prad_dataset import TcgaPradDataset
-
-
-@pytest.mark.skipif(not os.path.isdir(TCGA_PRAD_DATASET_DIR),
-                    reason="TCGA-PRAD dataset is unavailable")
-def test_dataset() -> None:
-    dataset = TcgaPradDataset(TCGA_PRAD_DATASET_DIR)
-
-    expected_length = 449
-    assert len(dataset) == expected_length
-
-    expected_num_positives = 10
-    assert dataset.dataset_df[dataset.LABEL_COLUMN].sum() == expected_num_positives
-
-    sample = dataset[0]
-    assert isinstance(sample, dict)
-
-    expected_keys = [dataset.SLIDE_ID_COLUMN, dataset.CASE_ID_COLUMN,
-                     dataset.IMAGE_COLUMN, dataset.LABEL_COLUMN]
-    assert all(key in sample for key in expected_keys)
-
-    image_path = sample[dataset.IMAGE_COLUMN]
-    assert isinstance(image_path, str)
-    assert os.path.isfile(image_path)
```

Comment:
To not leave things completely untested, do you think we could have a SlideDataset test? obviously we can't test the length or number of positives ... but we can test the dataset contains the expected keys and and that the content of the dict has the expected type. Looking at the dataset definition, if path is an existing path and we pass a dataset.csv, we can run these tests without need for mounting any real data. What you think?

Reply:
Now added a `test_slides_dataset.csv` and some basic tests in `test_slides_dataset.py`.